static/src/js/actions/project.js

```diff
@@ -338,131 +343,160 @@ export const getProjectCollections = () => async (dispatch, getState) => {
         }
       }
     }`
+  let response = null
 
-  const response = graphQlRequestObject.search(graphQuery, {
-    params: {
-      conceptId: filteredIds,
-      includeTags: defaultCmrSearchTags.join(','),
-      includeHasGranules,
-      pageSize: filteredIds.length
-    },
-    subcriptionParams: {
-      subscriberId: username
-    }
-  })
-    .then((responseObject) => {
-      const payload = []
+  try {
+    response = await graphQlRequestObject.search(graphQuery, {
```

Comment:
```suggestion
    const response = await graphQlRequestObject.search(graphQuery, {
```

Reply:
I set it as a let in line 346 and then return it at the end of everything